x-pack/plugin/relevance-search/src/main/java/org/elasticsearch/xpack/relevancesearch/relevance/curations/CurationsService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.relevancesearch.relevance.curations;
+
+import org.elasticsearch.client.internal.Client;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Manage relevance settings, retrieving and updating the corresponding documents in .ent-search index
+ */
+public class CurationsService {
+
+    public static final String ENT_SEARCH_INDEX = ".ent-search";
+    public static final String CURATIONS_SETTINGS_PREFIX = "curations-";
+    private static final String CONDITIONS_FIELD = "conditions";
+    private static final String PINNED_DOCS_FIELD = "pinned_document_ids";
+    private static final String EXCLUDED_DOCS_FIELD = "excluded_document_ids";
+    private static final String CONTEXT_ATTR = "context";
+    private static final String VALUE_ATTR = "value";
+    private static final String ID_ATTR = "_id";
+    private static final String INDEX_ATTR = "_index";
+
+    private final Client client;
+
+    public CurationsService(final Client client) {
+        this.client = client;
+    }
+
+    public CurationSettings getCurationsSettings(String curationId) throws CurationsSettingsNotFoundException {
+        // TODO cache relevance settings, including cache invalidation
+        final Map<String, Object> settingsContent = client.prepareGet(ENT_SEARCH_INDEX, CURATIONS_SETTINGS_PREFIX + curationId)
```

Comment:
I think what we want here is to query for the curations that belong to the given curations group.
the curation group is the one specified in the `relevance_match` query.

for example:

```
GET enterprise-search-engine-national-parks-demo/_search
{
  "query": {
    "relevance_match": {
       "query": "mountain",
       "curations": "parks-search-promos"
    }
  }
}
```
`parks-search-promos` is the group name.

and we could have multiple curations for the same group name:

```
PUT .ent-search/_doc/curations-parks-search-promos-1
{
   "type": "query_rule",
   "group_name": "parks-search-promos",
   "conditions": [
     { "context": "query", "value": "hike" }
   ],
   "pinned_document_ids": [
      ...
   ],
   "excluded_document_ids": [
    ...
   ]
}
```
and

```
PUT .ent-search/_doc/curations-parks-search-promos-2
{
   "type": "query_rule",
   "group_name": "parks-search-promos",
   "conditions": [
     { "context": "query", "value": "mountain" }
   ],
   "pinned_document_ids": [
     ...
   ],
   "excluded_document_ids": [
   ...
   ]
}
```
both curations belong to the same group, but the only the second curation will apply to given relevance_match query.